changelogs/config.yaml

```
@@ -0,0 +1,30 @@
+---
+changelog_filename_template: CHANGELOG.rst
```

```suggestion
changelog_filename_template: ../CHANGELOG.rst
```
This will *probably* place it one level up.

---

That seemed to work.